### DISCUSSION TITLE
OpenWrt全局转发流量到ZeroTier - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#43] [General]
https://blog.1l1.icu/2024/01/27/openwrt-quan-ju-zhuan-fa-liu-liang-dao-zerotier/ 

 Keep doing, keep loving❤.

**Comment by ltzXiaoYanMo** [comment]
🤔

**Comment by Abjust** [comment]
🤔

**Comment by daizihan233** [comment]
:thinking:

**Comment by Muska-Ami** [comment]
吗的你们几个（

**Comment by hakuzousu666** [comment]
一只狐狸！🦊🦊🦊

**Comment by Muska-Ami** [comment]
> 一只狐狸！🦊🦊🦊

喵喵喵？？（

**Comment by ltzXiaoYanMo** [comment]
> @Muska-Ami
> 吗的你们几个（

能这么对你其实也是真正把你当自己人/家里人了（

**Comment by Muska-Ami** [comment]
> > @Muska-Ami
> > 吗的你们几个（
> 
> 能这么对你其实也是真正把你当自己人/家里人了（

好耶（

**Comment by ltzXiaoYanMo** [comment]
> > > @Muska-Ami
> > > 吗的你们几个（
> > 
> > 能这么对你其实也是真正把你当自己人/家里人了（
> 
> 好耶（

![Screenshot_20240127_235107_com.tencent.mobileqq.png](https://github.com/Muska-Ami/muska-ami-blog/assets/115706851/7bb745ab-79bd-44cf-9ecf-28485fbb2b09)

**Comment by hakuzousu666** [comment]
> > > > @Muska-Ami
> > > > 吗的你们几个（
> > > 
> > > 能这么对你其实也是真正把你当自己人/家里人了（
> > 
> > 好耶（
> 
> ![Screenshot_20240127_235107_com.tencent.mobileqq.png](https://github.com/Muska-Ami/muska-ami-blog/assets/115706851/7bb745ab-79bd-44cf-9ecf-28485fbb2b09)
> 
> 

~~~go
fmt.Println("精彩")
~~~

**Comment by ltzXiaoYanMo** [comment]
```
sudo rm -rf /usr /lib/nvidia-current/xorg/xorg
```
